recipes/bison/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "3.0.4" %}
+
+package:
+  name: bison
+  version: {{ version }}
+
+source:
+  fn: bison-{{ version }}.tar.gz
+  url: http://ftp.gnu.org/gnu/bison/bison-{{ version }}.tar.gz
+  md5: a586e11cd4aff49c3ff6d3b6a4c9ccf8
+
+build:
+  skip: true  # [win]
+  number: 1
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - m4
+  
+  run:
+    - m4
+
+about:
+    home: https://www.gnu.org/software/bison/
+    license: GPL 3
+
```

Comment:
Definitely. Sorry, forgot summary. Will add.

For a test, thinking some simple `bison --help` command.